editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -833,7 +836,9 @@ void SpatialEditorViewport::_surface_focus_exit() {
 
 	view_menu->set_disable_shortcuts(true);
 }
-
+bool SpatialEditorViewport ::_is_node_locked(const Node *pnode) {
```

Comment:
This should be `p_node` (same in the header). We use the `p_` prefix for parameters, and `r_` for those passed as referenced and modified within the method.